src/monio/UtilsAtlas.cc

```diff
@@ -131,8 +131,9 @@ std::vector<size_t> createLfricAtlasMap(const std::vector<atlas::PointLonLat>& a
 atlas::Field getGlobalField(const atlas::Field& field) {
   if (field.metadata().get<bool>("global") == false) {
     atlas::array::DataType atlasType = field.datatype();
+    std::vector<atlas::idx_t> fieldShape = field.shape();
```

Comment:
Similarly if you only need the levels, could do `field.shape(consts::eVertical)`